topostats/filters.py

```diff
@@ -160,7 +160,7 @@ def flatten_image(self, image: np.ndarray, mask: np.ndarray = None) -> np.ndarra
         image = self.remove_tilt(image, mask)
         return image
 
-    def row_col_medians(self, image: np.array, mask: np.array = None) -> np.array:
+    def row_col_medians(self, image: np.ndarray, mask: np.ndarray = None) -> np.ndarray:
```

Comment:
TIL the difference between `array` and `ndarray`. Amazed it worked without this change.

Reply:
I know, I hadn't realised the [difference](https://stackoverflow.com/a/15879527) myself really.